docker-compose-services/newrelic/web-build/Dockerfile

```diff
@@ -0,0 +1,21 @@
+ARG BASE_IMAGE
+FROM $BASE_IMAGE
+
+# Install New Relic
+# @see https://docs.newrelic.com/docs/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent
+ARG NEW_RELIC_AGENT_VERSION
+ARG NEW_RELIC_LICENSE_KEY
+ARG NEW_RELIC_APPNAME
+
+RUN curl -L https://download.newrelic.com/php_agent/archive/${NEW_RELIC_AGENT_VERSION}/newrelic-php5-${NEW_RELIC_AGENT_VERSION}-linux.tar.gz | tar -C /tmp -zx \
+    && export NR_INSTALL_USE_CP_NOT_LN=1 \
+    && export NR_INSTALL_SILENT=1 \
+    && /tmp/newrelic-php5-${NEW_RELIC_AGENT_VERSION}-linux/newrelic-install install \
+    && rm -rf /tmp/newrelic-php5-* /tmp/nrinstall*
+
+RUN sed -i -e "s/REPLACE_WITH_REAL_KEY/${NEW_RELIC_LICENSE_KEY}/" \
+    -e "s/newrelic.appname[[:space:]]=[[:space:]].*/newrelic.appname=\"${NEW_RELIC_APPNAME}\"/" \
+    /etc/php/${PHP_DEFAULT_VERSION}/fpm/conf.d/newrelic.ini
+RUN sed -i -e "s/REPLACE_WITH_REAL_KEY/${NEW_RELIC_LICENSE_KEY}/" \
+    -e "s/newrelic.appname[[:space:]]=[[:space:]].*/newrelic.appname=\"${NEW_RELIC_APPNAME}\"/" \
+    /etc/php/${PHP_DEFAULT_VERSION}/cli/conf.d/newrelic.ini
```

Comment:
The technique used here copies only into /etc/php/${PHP_DEFAULT_VERSION}/*/conf.d - which means (currently) /etc/php/7.3/*/conf.d. But we might want php7.4 or something else. 

We can't use DDEV_PHP_VERSION here because it's determined at container start time. 

So that makes the previous suggestion more important. If you just put it in .ddev/php, it will go in the right place.

Alternately, copy into all the places (5.6/7.0/7.1/7.2/7.3/7.4/8.0) (I don't think 8.0 works yet?)